assets/migrations/mysql/004_add_authorization_model_pbdata.sql

```diff
@@ -0,0 +1,5 @@
+-- +goose Up
+ALTER TABLE authorization_model ADD COLUMN serialized_protobuf LONGBLOB;
```

Comment:
We're assuming for the scope and purposes of this work that there has been prior migrations to move from a model stored across multiple rows to a single row per authorization model. The database schema changes and data migrations we expect to have been completed prior to releasing this work include:

* Converging all rows for each model's type definitions into a single row with `(store_id, authorization_model_id, serialized_protobuf)`
* Modifying the PK to be `(store_id, authorization_model_id)` instead of `(store_id, authorization_model_id, type)`

The code here largely assumes the "future state" where we store a model as a single row. Other work streams will have to execute this migration process before we release the ABAC work officially.